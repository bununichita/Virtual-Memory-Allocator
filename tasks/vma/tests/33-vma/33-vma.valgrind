==4303== Memcheck, a memory error detector
==4303== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==4303== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==4303== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==4303== Parent PID: 4302
==4303== 
==4303== 
==4303== HEAP SUMMARY:
==4303==     in use at exit: 0 bytes in 0 blocks
==4303==   total heap usage: 12 allocs, 12 frees, 8,491 bytes allocated
==4303== 
==4303== All heap blocks were freed -- no leaks are possible
==4303== 
==4303== For lists of detected and suppressed errors, rerun with: -s
==4303== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
