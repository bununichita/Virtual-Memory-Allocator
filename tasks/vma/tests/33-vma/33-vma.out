Total memory: C8 bytes
Free memory: 100 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: A - 6E
Miniblock 1:	A	-	6E	| RW-
Block 1 end
DEALLOC_AREN
1
10
